spec/models/manageiq/providers/vmware/infra_manager/provision/cloning_spec.rb

```diff
@@ -1,51 +1,75 @@
 describe ManageIQ::Providers::Vmware::InfraManager::Provision::Cloning do
+  let(:zone)              { EvmSpecHelper.local_miq_server.zone }
+  let(:ems)               { FactoryBot.create(:ems_vmware_with_authentication, :zone => zone) }
+  let(:user_admin)        { FactoryBot.create(:user_admin) }
+  let(:template)          { FactoryBot.create(:template_vmware, :name => "template1", :ext_management_system => ems, :cpu_limit => -1, :cpu_reserve => 0) }
+  let(:provision)         { FactoryBot.create(:miq_provision_vmware, :userid => user_admin.userid, :miq_request => provision_request, :source => template, :request_type => 'template', :state => 'pending', :status => 'Ok', :options => options) }
+  let(:provision_request) { FactoryBot.create(:miq_provision_request, :requester => user_admin, :src_vm_id => template.id) }
+  let(:options) do
+    {
+      :pass          => 1,
+      :vm_name       => "clone test",
+      :number_of_vms => 1,
+      :cpu_limit     => -1,
+      :cpu_reserve   => 0,
+      :src_vm_id     => [template.id, template.name],
+    }
+  end
```

Comment:
NOTE I refactored this spec pulling out these options making it simpler to add a second context, I did this in a separate commit from adding the additional specs if it helps review: https://github.com/ManageIQ/manageiq-providers-vmware/pull/826/commits/7d6b7716e3941930e78122fd0f4883f100d70534